onnxruntime/core/graph/graph.cc

```diff
@@ -978,6 +959,7 @@ ADD_ATTR_IMPLS(TypeProto)
 #undef ADD_ATTR_LIST_IMPL
 #undef ADD_ATTR_IMPLS
 
+// TODO why isn't attr_name a const&?
```

Comment:
>attr_name a const&? [](http://example.com/codeflow?start=18&length=19)

This is passed by value so, at the caller's discretion it can either be copied or moved to the function arg.